src/components/ui/Merge/Bro/BBroMergeForm.tsx

```
@@ -154,15 +154,15 @@ export const BBroMergeForm = () => {
           <div className="flex-1 flex flex-col sm:flex-row items-center gap-3 ">
             <Button
               className="flex items-center justify-center gap-3 w-full md:w-auto md:h-[62px]"
-              disabled={!address || !amount}
+              disabled
               onClick={
                 isConnected
                   ? () => handleMerge()
                   : () => openConnectModal && openConnectModal()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)